operator/risc_zero/.gitignore

```diff
@@ -1,2 +1,2 @@
-lib/librisc_zero_verifier_ffi.a
-lib/librisc_zero_verifier_ffi.dylib
+lib/librisc_zero_verifier_ffi.so
+lib/librisc_zero_verifier.dylib
```

Comment:
were these changes necessary?

Reply:
I removed the `_ffi` suffix initially as I found it redundant. When compiling on linux the linker was unable to find the shared library for risc0 without having the `_ffi` suffix added which leads to the difference in naming. The I was not sure why we initially were ignoring the `.a` file so I changed to ignoring the `.so`. I don't have a strong opinion on the naming for the macos shared library.